### ISSUE TITLE
avifenc does not encode large 108 megapixel images

### ISSUE THREAD

**Issue by AlFmla** [Alkl58/MegaPixel#2]
avifenc does not encode large 108 megapixel images

**Comment by Alkl58** [comment]
Please elaborate what this issue has to do with this GUI? 
For me it sounds more like a technical limitation of the encoder itself.

**Comment by AlFmla** [comment]
Yes, I'm sorry, I don't know where to report this issue. Only encoding in [qavif-viewer](https://github.com/jj11hh/qavif-viewer) does not have this problem. What is the difference between avif encoders here and in qavif-viewer?

**Comment by Alkl58** [comment]
I successfully encoded a 100 megapixel image.
May I ask you witch codec/extension your picture has?

**Comment by AlFmla** [comment]
I understand the reason! avifenc requires a lot of RAM for large files, especially with Color Format 444. I have 16GB of RAM, not enough even with Color Format 400 to encode 108 megapixel images! You need to set Color Format to 420 by default so that people can encode large files with little RAM. Can you report this issue to the avifenc developers? qavif-viewer only uses 2.5GB of memory to encode 108 megapixel images, how?

**Comment by AlFmla** [comment]
avifenc uses only AOM codec. Can you, or anyone else, Building avifenc with AVIF_CODEC_RAV1E? I don't know how. It would be very interesting to compare codecs. And perhaps the problem with large files would have been solved.
